lesson01/assignments/scripts.js

```diff
@@ -1,26 +1,40 @@
 // jQuery Doc Ready
 $(function () {
     // Set up some data and variables
-    const data = [1, 2, 3, 4],
-        input = $('.input'),
-        submit = $('.submit'),
-        resultSum = $('.sum');
+    let data = [5, 10, 15, 20],
+        submit = $('.submit');
+        values = $('h3');
+    // Print original array    
+    values.text(values.text() + data);
     // Click event on the submit input
     submit.on('click', function () {
```

Comment:
Instead of only a single submit event listener, which finds correct input value and method to apply based on id of submit button clicked, why not have a separate event listener and selectors for each of the functions? Something like: 
```
            <h2>Try every</h2>
            <!-- Input -->
            <input type="text" class="input every">
            <!-- Submit for Input -->
            <input id="every" class="submit" type="submit"></input>
            <!-- Result markup with hook for answer -->
            <p class="markup every">All my elements are less than the input: </p>
```

```
$('#every').on('click', function () {
  const val = $('.every.input').val()
  // etc
  $('.every.markup').text(tempText);
}
```

This would be more code, but easier to read and debug. The logic as it is now is a little fragile, dependent on the HTML elements not changing position in the document.